# Timestamp and sequence number
Header header

#vector of points of the trajectory
droneRefCommand[] 	droneTrajectory
bool  			is_periodic
int32 			initial_checkpoint # usually 1
